test/verify/machineslib.py

```diff
@@ -513,6 +514,7 @@ def checkLibvirtEnabled():
 
         # Make sure that unprivileged users can see the VM list when libvirtd is not running
         m.execute("systemctl stop libvirtd.service")
+        self.addCleanup(m.execute, "systemctl start libvirtd.service || true")
```

Comment:
Did you see that failing anywhere? If it does, wouldn't that break the next test anyway?

Reply:
I don't remember now. I can try to remove the `true` and see.